### PR TITLE
docs: [ovh sd] Added missing label for OVH dedicated server in SD

### PR DESCRIPTION
**Pull request by jiekun**
`__meta_ovhcloud_dedicated_server_no_intervention` label is available for OVH Cloud dedicated server. But it's missing in doc.

See: https://github.com/prometheus/prometheus/blob/da1b7a17b31932e56716c4e3dfc275b66bf3ba8a/discovery/ovhcloud/dedicated_server.go#L149

And previous test case: 
https://github.com/prometheus/prometheus/blob/da1b7a17b31932e56716c4e3dfc275b66bf3ba8a/discovery/ovhcloud/dedicated_server_test.go#L65

### CODE REVIEW

**Comment by jiekun** [comment]
@machine424 @roidelapluie Hi May I have your review of this PR? Thanks!

File: docs/configuration/configuration.md [hunk 1]
```diff
@@ -1467,6 +1467,7 @@ For OVHcloud's [public cloud instances](https://www.ovhcloud.com/en/public-cloud
 * `__meta_ovhcloud_dedicated_server_ipv6`: the IPv6 of the server
 * `__meta_ovhcloud_dedicated_server_link_speed`: the link speed of the server
 * `__meta_ovhcloud_dedicated_server_name`: the name of the server
+* `__meta_ovhcloud_dedicated_server_no_intervention`: the [intervention](https://support.us.ovhcloud.com/hc/en-us/articles/27991435200147-FAQ-Interventions-and-Hardware-Replacement) of the server.
```
Comment (machine424) [inline docs/configuration/configuration.md:1470]: Maybe we should precise whether `no_intervention` means:
- interventions are disabled on the server.
OR
- there is no interventions currently on the server.

Comment (jiekun) [reply docs/configuration/configuration.md:1470]: Thank you for the review and sorry for my late reply, I was out of the office yesterday.

According to OVH's [document]( https://eu.api.ovh.com/console-preview/?section=%2Fdedicated%2Fserver&branch=v1#get-/dedicated/server/-serviceName-), `noIntervation` refer to:
- Prevent datacenter intervention. 

So in this case, it means "interventions are disabled on the server". And I'd like to suggest using the following description:
- `is this server immune to datacenter intervention`.

WDYT @machine424

File: docs/configuration/configuration.md [hunk 2]
```diff
@@ -1467,6 +1467,7 @@ For OVHcloud's [public cloud instances](https://www.ovhcloud.com/en/public-cloud
 * `__meta_ovhcloud_dedicated_server_ipv6`: the IPv6 of the server
 * `__meta_ovhcloud_dedicated_server_link_speed`: the link speed of the server
 * `__meta_ovhcloud_dedicated_server_name`: the name of the server
+* `__meta_ovhcloud_dedicated_server_no_intervention`: is this server immune to datacenter intervention. Also see: [noIntervention](https://eu.api.ovh.com/console-preview/?section=%2Fdedicated%2Fserver&branch=v1#get-/dedicated/server/-serviceName-)
```
Comment (machine424) [inline docs/configuration/configuration.md:1470]:
```suggestion
* `__meta_ovhcloud_dedicated_server_no_intervention`: whether datacenter intervention is disabled.
```
WDYT?
I see, we already link the API above (users can look for details there.)

Comment (jiekun) [reply docs/configuration/configuration.md]: This one looks better. Thank you.

**Review by machine424 — APPROVED** [review]
thanks.

**Comment by machine424** [comment]
For the failing DCO job:

Thanks for your PR. For legal reasons, we require that all commits are signed with a [DCO](https://developercertificate.org/) before we can merge them. See [this blog post](https://www.cncf.io/blog/2016/11/08/cloud-native-software-can-trust/) for considerations around this.

This means that the last line of your commit message should read like:

`Signed-Off-By: Your Name <your@email.address>`

If you are using GitHub through the web interface, it's quickest to close this PR and open a new one with the appropriate line.

If you are using Git on the command line, it is probably quickest to amend and force push. You can do that with
````
git commit --amend --signoff
git push -f $remote $remote_branch_for_pr
````

As always, be careful when force-pushing.

_Originally posted by @roidelapluie_

**Comment by jiekun** [comment]
@machine424 Thanks again. I just fixed it. And sorry for the trouble.

**Comment by roidelapluie** [comment]
Thanks!

**Pull request merged**